docs/vercel.json

```diff
@@ -1,11 +1,6 @@
 {
   "cleanUrls": true,
   "redirects": [
```

Comment:
Do you mean the old URLs from `/get-started/*`? Or some other URLs?

The URLs that were previously under Platform are still under `/platform/*`.

Reply:
We should try to preserve old links (or add redirects) as much as possible, especially since we have many websites linking to these tutorials and we don't want to show them a 404 or lose the link equity: e.g. this [stackoverflow question](https://stackoverflow.com/questions/71526371/nhost-typeerror-intermediate-value-is-not-a-function) links to the following page: https://docs.nhost.io/get-started/quick-start/javascript-client which would show a 404 with this PR.

Reply:
Added redirects for the CLI URLs. I'll add redirects for the quickstart guide and migrations tutorial once this PR is merged and the two blog posts are up.